### ISSUE TITLE
[iOS] Spinner indicator inside the search field behaves weirdly while dragging around the map

### ISSUE THREAD

**Issue by beyondcompute** [organicmaps/organicmaps#683]
See the following video, upper left corner.

https://user-images.githubusercontent.com/248055/122777212-7ee7d180-d2b4-11eb-923f-8d0d985cbec9.mov

**Comment by Jean-BaptisteC** [comment]
Can you reupload video ? :)

**Comment by Jean-BaptisteC** [comment]
@matheusgomesms can you on latest version?

**Comment by matheusgomesms** [comment]
Yes, still there. It happens only when you search, and click on the lower button “see on the map”. So when you move the map, the displayed results are updated (because again, you are looking for something on the map not selected yet).

if you make a search and click on the result, this doesn’t happen.
